src/plugins/github/declaration.js

```diff
@@ -113,7 +129,14 @@ const mentionsAuthorEdgeType = Object.freeze({
   // TODO(#811): Probably change this to 0
   defaultBackwardWeight: 1 / 32,
   prefix: E.Prefix.mentionsAuthor,
-  description: "TODO: Add a description for this EdgeType",
+  description: dedent`\
+    When a post in a particular thread mentions a particular user, that post
+    will have a \`MentionsAuthorEdge\` connecting it to any posts written by the
+    mentioned user. The intuition is that if a post is mentioning an author by name,
+    their contributions in that thread are probably particularly valuable.
+
+    This is an experimental feature and may be removed in a future version of SourceCred.
+  `,
```

Comment:
Maybe just revisit this description.. the word 'particular' is repeated 3x. 

"When a post in a thread mentions a user, that post will have a \`MentionsAuthorEdge\` connecting it to every post by that user in that particular thread". Idk if that's any clearer.